instrumentation/jmx-metrics/library/src/main/java/io/opentelemetry/instrumentation/jmx/yaml/MetricStructure.java

```diff
@@ -20,24 +20,39 @@
  * <li>the metric attributes
  * <li>the unit
  *
- *     <p>Known subclasses are JmxRule and Metric.
+ *     <p>Known subclasses are {@link JmxRule} and {@link Metric}.
  */
 abstract class MetricStructure {
 
   // Used by the YAML parser
+  //
   //    type: TYPE
   //    metricAttribute:
   //      KEY1: SPECIFICATION1
   //      KEY2: SPECIFICATION2
   //    unit: UNIT
-
-  private Map<String, String> metricAttribute; // unused, for YAML parser only
+  //
+  // For state metrics
+  //
+  //    type: state
+  //    metricAttribute:
+  //      KEY1: SPECIFICATION1
+  //      state:
+  //        state1: [a,b]
+  //        state2: c
+  //        state3: '*'
```

Comment:
Should this use _ instead of *?
```suggestion
  //        state3: _
```